pom.xml

```diff
@@ -379,4 +423,94 @@
             </build>
         </profile>
     </profiles>
+
+    <repositories>
+
+        <repository>
+            <id>maven.central</id>
+            <name>Maven Central</name>
+            <url>https://repo1.maven.org/maven2</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+            </releases>
+        </repository>
+
+
+        <repository>
+            <id>redhat.ga</id>
+            <name>Red Hat General Availability Repository</name>
+            <url>https://maven.repository.redhat.com/ga</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+            </releases>
+        </repository>
+
+
+        <repository>
+            <id>redhat.ea</id>
+            <name>Red Hat Early Access Repository</name>
+            <url>https://maven.repository.redhat.com/earlyaccess/all</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+            </releases>
+        </repository>
+
+    </repositories>
+
+    <pluginRepositories>
+
+        <pluginRepository>
+            <id>maven.central</id>
+            <name>Maven Central</name>
+            <url>https://repo1.maven.org/maven2</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+            </releases>
+        </pluginRepository>
+
+
+        <pluginRepository>
+            <id>redhat.ga</id>
+            <name>Red Hat General Availability Repository</name>
+            <url>https://maven.repository.redhat.com/ga</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+            </releases>
+        </pluginRepository>
+
+
+        <pluginRepository>
+            <id>redhat.ea</id>
+            <name>Red Hat Early Access Repository</name>
+            <url>https://maven.repository.redhat.com/earlyaccess/all</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+            </releases>
+        </pluginRepository>
+
+    </pluginRepositories>
```

Comment:
I've removed this block of code from pom.xml and it worked without any problem in Travis. Here is the Travis build https://travis-ci.org/carlosthe19916/xavier-integration/jobs/625773720 and here is the GitHub commit https://github.com/carlosthe19916/xavier-integration/commit/3c9433adc2498f3df73c78fe2c88ace33c009d25 ...  I wonder if we can remove this block? What do you think?

Reply:
It was working before in Travis with that block. What's the issue with adding the repositories ? 
If there is no artifact that comes from those repositories then I agree we dont need it.

Reply:
In fact, these are the repositories we are using, but we are using a customized settings.xml file for travis..... I think we should move away from having things in the settings.xml and put it declaratively in our pom

Reply:
Also, for the github Actions we need to have these repositories , either following the same approach as travis or CI agnostic putting them in the pom
It's true that this change is related to the Github migration task and not to the End2End